command/agent/dns_test.go

```diff
@@ -1986,8 +1987,8 @@ func TestDNS_ServiceLookup_LargeResponses(t *testing.T) {
 
 	longServiceName := "this-is-a-very-very-very-very-very-long-name-for-a-service"
 
-	// Register 3 nodes
-	for i := 0; i < 3; i++ {
+	// Register a lot of nodes.
+	for i := 0; i < 4; i++ {
```

Comment:
I wanted this to hit the initial trim based on the number of entries, and then hit the size-based trim as well. Just to make it go through more of the code.